config/update_config.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 
+# make sure user is root
```

Comment:
if we ever set up an automated system for deploying changes, we'll want to be careful with automating any config changes. I like the idea of requiring root. Perhaps all config changes should always be done manually

Reply:
agree with this!